src/pages/dashboard/game/[id].tsx

```diff
@@ -0,0 +1,112 @@
+import styles from "../main.module.scss";
+import { Layout } from "src/components/global/layout.module";
+import { Header } from "@components/dashboard/header";
+import { Accordion } from "@components/dashboard/accordion";
+import Prisma from "@prisma/client";
+import { trpc } from "@utils/trpc";
+import { GetStaticPaths, GetStaticProps, NextPage } from "next";
+import { Addon } from "@components/setup/addon.module";
+import { prisma } from "src/server/prisma";
+import { Button } from "@components/input/button.module";
+import { Toggle } from "@components/input/toggle";
+import { useState } from "react";
+import { User } from "@components/dashboard/user";
+
+import Marceline from "@public/avatars/marceline.svg";
+import Ghost from "@public/avatars/ghost.svg";
+import Frog from "@public/avatars/froggi.svg";
+
+const GameDashboard: NextPage<Props> = ({ game }) => {
+  const addons = trpc.addon.all.useQuery({ limit: 5, game_id: game.id });
+  const [allowNsfw, setAllowNsfw] = useState(true);
+  console.log(allowNsfw);
+
+
+  return (
+    <Layout>
+      <div className={styles.main}>
+        <Header
+          title={game.title}
+          description={game.description}
+          avatar={Marceline}
+        />
+
+        <main className={styles.menu}>
+          <Accordion title="Statistics">
+            a
+          </Accordion>
+
+          <Accordion title="Default Addons">
+            <section className={styles.itemList}>
+              {addons.data?.items.map(addon => <Addon key={addon.id} addon={addon} />)}
+              <Button variant="secondary">Add default addon</Button>
+            </section>
+          </Accordion>
+
+          <Accordion title="Default Settings">
+            <section className={styles.settings}>
+              <Toggle
+                name="allow_nsfw"
+                label="Allow nsfw"
+                value={allowNsfw}
+                onChange={() => setAllowNsfw(old => !old)}
```

Comment:
Should probably extract this to a custom `useBoolean` hook, since we use this boolean toggle state in more places

Example: https://usehooks-ts.com/react-hook/use-boolean